src/Worker.Extensions.DurableTask/Worker.Extensions.DurableTask.csproj

```diff
@@ -29,7 +29,7 @@
     <AssemblyOriginatorKeyFile>..\..\sign.snk</AssemblyOriginatorKeyFile>
 
     <!-- Version information -->
-    <VersionPrefix>1.5.0</VersionPrefix>
+    <VersionPrefix>1.5.1</VersionPrefix>
```

Comment:
Since we're adding new public APIs, this should be a minor version update. However, it looks like the latest version is still `1.4.0`, so I think we can leave this as `1.5.0`.